lib/views/widgets/progress_bar.dart

```diff
@@ -0,0 +1,144 @@
+import 'package:flutter/material.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+import 'package:sun_flutter_capstone/consts/global_style.dart';
+
+class ProgressBar extends StatefulHookConsumerWidget {
+  double progress;
```

Comment:
Is there a reason where this should not be final
```suggestion
  final double progress;
```

Reply:
Logically it doesn't make sense since it can change but it can be final